File: app/src/main/java/com/example/sunddenfix/retrofit/utils/IOUtil.java
package com.maogu.htclibrary.util;

import java.io.Closeable;
import java.io.IOException;

/**
 * IO工具
 */
public class IOUtil {

    /**
     * 关闭流
     *
     * @param stream 可关闭的流
     */
    public static void closeStream(Closeable stream) {
        try {
            if (stream != null) {
                stream.close();
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/com/example/sunddenfix/retrofit/utils/IOUtil.kt
package com.example.sunddenfix.retrofit.utils

import java.io.Closeable

class IOUtil {
    companion object {
        fun closeStream(stream: Closeable) {
            try {
                stream.close()
            } catch (e: Exception) {
                e.printStackTrace()
            }
        }
    }
}